example-web/README.md

```diff
@@ -0,0 +1,26 @@
+# Web Example
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Setup
+
+This example project can be used to test/demo this project in the context of
+`react-native-web`. Here's how to use it:
+
+1. Build the library:
+    ```sh
+    # go to the `package` folder
+    cd ../package
+    # build the library
+    yarn prepare
```

Comment:
I can see this PR delivers both `package-lock.json` as well as `yarn.lock`.
In this project we use npm as it is more strict and we want to make sure that the package installs correctly using it.
Would it be possible to change it to `npm run prepare` and remove `yarn.lock`, or maybe there are reasons behind having `yarn` here?